Source/CesiumRuntime/Public/CesiumFeatureIdAttribute.h

```diff
@@ -131,27 +131,27 @@ class CESIUMRUNTIME_API UCesiumFeatureIdAttributeBlueprintLibrary
       UPARAM(ref) const FCesiumFeatureIdAttribute& FeatureIDAttribute);
 
   /**
-   * Get the number of vertices in the primitive containing the feature
-   * ID attribute. If the feature ID attribute is invalid, this returns 0.
+   * Get the number of feature IDs in this attribute. If the feature ID
+   * attribute is invalid, this returns 0.
    */
   UFUNCTION(
       BlueprintCallable,
       BlueprintPure,
       Category = "Cesium|Features|FeatureIDAttribute")
   static int64
-  GetVertexCount(UPARAM(ref)
-                     const FCesiumFeatureIdAttribute& FeatureIDAttribute);
+  GetFeatureIDCount(UPARAM(ref)
```

Comment:
This might be a bit confusing on second glance -- I don't want this to convey the # of actual unique feature IDs versus. the length of the attribute. Let me tweak this

Reply:
Settled on `GetCount` with some additional documentation. Open to other alternatives too.